src/component/tooltip/TooltipView.ts

```diff
@@ -257,6 +257,7 @@ class TooltipView extends ComponentView {
             // self.manuallyShowTip({x, y}) might cause tooltip hide,
             // which is not expected.
             && tooltipModel.get('triggerOn') !== 'none'
+            && tooltipModel.get('triggerOn') !== 'click'
```

Comment:
It will be better to cache the `tooltipModel.get('triggerOn')` to a temporal variable `triggerOn` if there are multiple accesses.